modules/services/preferences/src/main/java/au/com/shiftyjelly/pocketcasts/preferences/SettingsImpl.kt

```diff
@@ -999,7 +1001,17 @@ class SettingsImpl @Inject constructor(
     }
 
     override fun getAutoPlayNextEpisodeOnEmpty(): Boolean {
-        return getBoolean(Settings.PREFERENCE_AUTO_PLAY_ON_EMPTY, false)
+        val defaultValue = when (Util.getAppPlatform(context)) {
+            AppPlatform.Automotive -> true
+            AppPlatform.Phone -> true
+            AppPlatform.WearOs -> false
```

Comment:
Leaving this feature off in wear os because I haven't added the setting on wear os. I'm not sure how many settings we want to add to wear os, so I'm holding off on this for now and leaving the feature off, but I'm sure this is something we'll revisit later.